app/backend/src/couchers/servicers/conversations.py

```diff
@@ -12,8 +12,10 @@
 from couchers.utils import Timestamp_from_datetime
 from pb import api_pb2, conversations_pb2, conversations_pb2_grpc
 
-# TODO: custom pagination length
+# TODO: Still needs custom pagination: GetUpdates
 PAGINATION_LENGTH = 20
+DEFAULT_LARGE_PAGINATION_LENGTH = 20
+DEFAULT_SMALL_PAGINATION_LENGTH = 10
```

Comment:
I thought it makes sense to have a larger default for messages than chats

Reply:
Yeah, that's sensible, though maybe we could just do it on the frontend and set some reasonable maximum here.

Also, maybe worth either renaming to `DEFAULT_{MESSAGES/CONVERSATIONS}_PAGE_SIZE` or adding a comment as to when large/small is used?